pyproject.toml

```diff
@@ -225,7 +225,7 @@ exclude_lines = [
 ]
 
 [tool.pytest.ini_options]
-log_cli = true
+log_cli = false
```

Comment:
:+1:  this should avoid at least a few weirdly interleaved outputs as well

Reply:
That affects the log level of pytest in Pycharm as well 😬 . We can overwrite this configuration using `--log-cli-level=DEBUG -s` in the pytest test configuration (template). The `-s` disables all capturing and should help with interleaved outputs.